### DISCUSSION TITLE
Ask some experience in fine-tuning the dataset construction.

### DISCUSSION THREAD

**Discussion by jizh18** [fishaudio/fish-speech#519] [Q&A]
好兄弟们，请教大家几个问题，太感谢了
我们现在希望将这个方法迁移到乌干达的小语种上，想问问您怎么去构建数据集会比较好，有这么几个具体的问题。
1. 数据集的平均长度每一条多长比较合适。
2. 数据集的lab文件需要对音频的起始和终止时间进行准确标记嘛。
3. 如果这个语言和英语非常像，只是一些发音上的变化，需要大概多少个小时的数据集会比较合适。

      此致
敬礼

**Comment by Stardust-minus** [comment]
Move to Disscusion and change title to English.
